resources/manifests/pod-checkpointer-psp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: pod-checkpointer-restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+  # Allow core volume types.
+  volumes:
+  - 'configMap'
+  - 'hostPath'
+  - 'secret'
+  hostNetwork: true
```

Comment:
Not sure why we need this. But if it is needed... :)

Reply:
As I said, it's copied from Pod definition, so we should check there instead.

Reply:
Oh, if it's there... it is probably needed. We can open an issue to double check this later :)

Reply:
Taking note to create an issue and see if we can improve it. Out of scope for this PR.